backend/src/lira_importing/readme.md

```diff
@@ -0,0 +1,82 @@
+# readme
+
+**These functions are only meant to be used ones to import the lira database, reformat it and uploaded to our own database. For these scripts to work you need a copy off the db.js and liradb.js for the scripts to work. These files will not be uploaded to github.**
```

Comment:
Could you add a `db.example.js` and `liradb.example.js` with fake password/user plz?

Reply:
Also could you add a small subsection where you describe which data is imported from lira? It doesn't need to big a huge paragraph but at least the column could be great

Reply:
btw to fix prettier, in the `backend` folder you can run `npx prettier -w /src/lira_importing/`

Reply:
@s203860 the explanation was for the sys spec (add a 5.2.4 section), we can keep it here but we need it in the sys spec too. Also could you check 5.2.1 and 5.2.3 and update given Ekkart comments plz?

Reply:
(i finishing something for valhalla and then i will take a look)